src/kakarot/instructions/environmental_information.cairo

```diff
@@ -256,6 +270,9 @@ namespace EnvironmentalInformation {
         let is_warm = State.is_account_warm(evm_address);
         tempvar gas = is_warm * Gas.WARM_ACCESS + (1 - is_warm) * Gas.COLD_ACCOUNT_ACCESS;
         let evm = EVM.charge_gas(evm, gas);
+        if (evm.reverted != FALSE) {
```

Comment:
same question as for balance

Reply:
well if we charge gas for the operation but there's not enough gas left it's supposed to revert with OOG. It probably caused a bug in case code for tests supposed to be OOG when doing cold accesses